documentation/user_docs/docs/manual/Binary_operations.rst

```diff
@@ -2,110 +2,154 @@
+mrdivide``, ``mldivide``, etc.).
 
-There are several options for the input parameters to binary operations
+sqw objects
+===========
 
+Let us take for our example the addition operator ``+``, and our initial ``sqw`` object is called ``w1`` and has an
+attached ``pix`` array.
 
-sqw objects
------------
+.. note::
 
-Let us take for our example the addition operator '+', and our initial single sqw object is called ``w1`` and has the 'pix' array retained (note this is an important point - you can have an sqw object without the pix array by converting a dnd object to sqw. In that case the result is referred to as an sqw object of dnd type. In our example ``w1`` has a pix array, so is referred to as an sqw of sqw type).
+   In the case of ``sqw`` objects, binary operations transform the ``signal`` and ``variance`` elements of the pixel data.
+..
+   .. note::
+
+      You can have an ``sqw`` object without the pix array by converting a ``dnd`` to ``sqw``. Though this is inadvisable.
 
 Single sqw object
-=================
+-----------------
 
-You can add to w1 in the following way
+You can add values to ``w1`` in the following ways:
 
-::
+- ``w2_sqw`` is an ``sqw`` object with a pix array of identical size to the pix array of ``w1``.
+
+  ::
+
+     wout = w1 + w2_sqw;
+
+.. note::
 
-   wout = w1 + w2_sqw
-   wout = w1 + w2_sqw_dnd_type
-   wout = w1 + w2_dnd
-   wout = w1 + scalar
+   A common use for this is when a background dataset has been created that maps exactly onto the real dataset, and
+   needs to be subtracted.
 
+- ``w2_dnd`` is a ``dnd`` object commensurate with ``w1.data``
 
-The conditions for these operations are as follows:
+  ::
 
-- ``w2_sqw`` is an sqw of sqw type, with a pix array of identical size to the pix array of ``w1``. There is only really one circumstance in which doing a binary operation of this type makes sense - when a background dataset has been created that maps exactly onto the real dataset, and needs to be subtracted.
+     wout = w1 + w2_dnd;
 
-- ``w2_sqw_dnd_type`` is an sqw of dnd type (i.e. no it has pix array) whose plot axes overlap exactly with those of ``w1``. An example is taking a 1d cut along the energy axis from two different regions of reciprocal space, and then adding or subtracting one from the other. In this case the output will be a sqw object of dnd type, since the pixel information has lost its connection with the signal and error that are plottable.
 
-- ``w2_dnd`` as above, but ``w2_dnd`` is a dnd object rather than an sqw of dnd type. Similarly to the above, the output is an sqw object of dnd type.
+- ``num`` is a numeric array of the same size as the pix array of ``w1``.
 
-- ``scalar`` is a single number, e.g. if you want to add 4.782 to all of the data in ``w1``. The output stays as a full sqw of sqw type (with pix array).
+  ::
+
+     wout = w1 + num;
+
+- ``scalar`` is a single number, e.g. if you want to add 4.782 to all of the data in ``w1``.
```

Comment:
```suggestion
- by adding a single number (here ``scalar``) to all of the data in ``w1``.
```
Your example of 4.782 is good but in that case you should use that number, not ``scalar``.